.github/workflows/ci.yml

```diff
@@ -630,3 +630,105 @@ jobs:
           notify_when: 'failure'
         env:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_NOTIFY_BUILD }}
+
+  build-and-publish-forc-binary:
+    name: Forc Binary - ${{ matrix.job.target }} (${{ matrix.job.os }})
+    runs-on: ${{ matrix.job.os }}
+    if: github.event_name == 'release' && github.event.action == 'published'
+    needs: [
+      cancel-previous-runs,
+      publish,
+    ]
+    strategy:
+      matrix:
+        job:
+          - os: ubuntu-latest
+            platform: linux
+            target: x86_64-unknown-linux-gnu
+            arch: amd64
+            svm_target_platform: linux-amd64
+          - os: ubuntu-latest
+            platform: linux
+            target: aarch64-unknown-linux-gnu
+            arch: arm64
+            svm_target_platform: linux-aarch64
+          - os: macos-latest
+            platform: darwin
+            target: x86_64-apple-darwin
+            arch: amd64
+            svm_target_platform: macosx-amd64
+          - os: macos-latest
+            platform: darwin
+            target: aarch64-apple-darwin
+            arch: arm64
+            svm_target_platform: macosx-aarch64
+          # FIXME: Have no idea what to do on a Windows machine <(-_-<)
```

Comment:
@JoshuaBatty 
- So I'm unashamedly definitely not the go-to guy for Windows-related stuff. I don't think I've ever worked in any capacity on a Windows machine. Would I tag this with a "help-needed" label, or reach out to someone in the tooling chat for help with this? 
- Everything else works as expected

Reply:
@ra0x3 I've opened #1526 with some context on the current Windows sitch - I think until that issue is addressed we should leave out the Windows build for now.